src/Symfony/Component/Uid/composer.json

```diff
@@ -23,6 +23,12 @@
         "php": ">=7.2.5",
         "symfony/polyfill-uuid": "^1.15"
     },
+    "require-dev": {
+        "symfony/console": "^4.4|^5.0"
+    },
+    "suggest": {
+        "symfony/console": "For using the console commands"
+    },
```

Comment:
AFAIK we don't add `suggest` anymore 🤔

Reply:
I will wait for a confirmation before removing it.

Reply:
Oskar is correct, we do not add new `suggest` entries.